docs/command/osdctl_account.md

```diff
@@ -34,7 +34,7 @@ osdctl account [flags]
 * [osdctl account clean-velero-snapshots](osdctl_account_clean-velero-snapshots.md)	 - Cleans up S3 buckets whose name start with managed-velero
 * [osdctl account cli](osdctl_account_cli.md)	 - Generate temporary AWS CLI credentials on demand
 * [osdctl account console](osdctl_account_console.md)	 - Generate an AWS console URL on the fly
-* [osdctl account get](osdctl_account_get.md)	 - get resources
+* [osdctl account get](osdctl_account_get.md)	 - Get resources
```

Comment:
Is this related?

Reply:
Umm, not related. I can remove all the unrelated changes if you think that's better. @fahlmant